### DISCUSSION TITLE
Visibilidade do Repositório

### DISCUSSION THREAD

**Discussion by psantos10** [ilhas-maldivas/como-sair-de-angola#2] [General]
Boas 

Vi agora que o repositório é privado. Não era suposto ser público?

**Comment by e200** [comment]
Será privado até decidirmos que o conteúdo está pronto para pré-lançamento.

**Comment by hkakutalua** [comment]
> Boas 
> 
> Vi agora que o repositório é privado. Não era suposto ser público?
> 

Não sei o que se alterou por conta das discussões no grupo, mas a ideia foi sempre ter essas informações restritas só para os membros do grupo.

**Reply by psantos10** [reply]
> > Boas
> > Vi agora que o repositório é privado. Não era suposto ser público?
> 
> Não sei o que se alterou por conta das discussões no grupo, mas a ideia foi sempre ter essas informações restritas só para os membros do grupo.

Quem terá a responsabilidade de adicionar novos membros na organização do github de forma a terem acesso ao repositório, uma vez que faz parte do grupo? E como será feita a manutenção para aqueles casos em que o membro é removido do grupo por não participar?

**Reply by e200** [reply]
As informações sensíveis do grupo ficam no grupo. Mas aquelas que concordamos em partilhar são disponibilizadas aqui publicamente.

**Reply by hkakutalua** [reply]
> > > Boas
> > > Vi agora que o repositório é privado. Não era suposto ser público?
> > 
> > Não sei o que se alterou por conta das discussões no grupo, mas a ideia foi sempre ter essas informações restritas só para os membros do grupo.
> 
> Quem terá a responsabilidade de adicionar novos membros na organização do github de forma a terem acesso ao repositório, uma vez que faz parte do grupo? E como será feita a manutenção para aqueles casos em que o membro é removido do grupo por não participar? 

Bom problema pra criar bot 😁.

**Reply by e200** [reply]
> > > Boas
> > > Vi agora que o repositório é privado. Não era suposto ser público?
> > 
> > 
> > Não sei o que se alterou por conta das discussões no grupo, mas a ideia foi sempre ter essas informações restritas só para os membros do grupo.
> 
> Quem terá a responsabilidade de adicionar novos membros na organização do github de forma a terem acesso ao repositório, uma vez que faz parte do grupo? E como será feita a manutenção para aqueles casos em que o membro é removido do grupo por não participar?

Esta organização funciona com uma certa independência do grupo, para contribuir não precisa ser membro.
